### ISSUE TITLE
is this project still applicable

### ISSUE THREAD

**Issue by nebulousdog** [mochajs/eslint-config-mocha#1]
this project looks dead? should the readme point to https://github.com/lo1tuma/eslint-plugin-mocha?

**Comment by basickarl** [comment]
Was just about to ask the same question. @boneskull  ?

**Comment by boneskull** [comment]
no, this repo was originally intended for use by projects in the mochajs org, but is currently unused.
